### DISCUSSION TITLE
[CLOSED] Deployer integration

### DISCUSSION THREAD

**Discussion by htahir1** [zenml-io/zenml#215] [Show and tell]
A pivotal part of any ML pipeline is the deployment step. We plan to support many libraries to help automatically deploy models, but we need your help to decide which one to start from:

* [BentoML](https://github.com/bentoml/BentoML) > Vote with :heart: 
* [KServe](https://github.com/kserve/kserve) > Vote with :smile: 
* [Ray Serve](https://docs.ray.io/en/latest/serve/index.html) > Vote with :tada:
* [MLFlow Deployer](https://www.mlflow.org/docs/latest/python_api/mlflow.deployments.html) > Vote with :rocket: 
* [Seldon Core](https://github.com/SeldonIO/seldon-core) > Vote with 👀 

Get your votes in before January 30th 2022!

**Comment by withsmilo** [comment]
How about [Seldon](https://github.com/SeldonIO/seldon-core)?

**Reply by htahir1** [reply]
Good item to have on the list! I'll update
